packages/aws-cdk/lib/api/cxapp/cloud-executable.ts

```diff
@@ -48,8 +48,8 @@ export class CloudExecutable {
   /**
    * Synthesize a set of stacks
    */
-  public async synthesize(): Promise<CloudAssembly> {
-    if (!this._cloudAssembly) {
+  public async synthesize(cacheCloudAssembly: boolean = true): Promise<CloudAssembly> {
```

Comment:
add docstring for `cacheCloudAssembly`. I couldn't understand what this means

Reply:
👍